Documentation/spelling_wordlist.txt

```diff
@@ -480,6 +481,7 @@ Sep
 setrlimit
 sfc
 sgr
+sha
```

Comment:
I do not see where this is added in the documentation changes, and I don't think it should be added; ideally "SHA" should be used instead of "sha".

Reply:
The doc build was already broken. It was added as part of a PR that didn't run doc tests.